modules/core/www/no_cookie.php

```diff
@@ -11,8 +11,15 @@
 $t = new \SimpleSAML\XHTML\Template($globalConfig, 'core:no_cookie.tpl.php');
 $translator = $t->getTranslator();
 
-$t->data['header'] = htmlspecialchars($translator->t('{core:no_cookie:header}'));
-$t->data['description'] = htmlspecialchars($translator->t('{core:no_cookie:description}'));
-$t->data['retry'] = htmlspecialchars($translator->t('{core:no_cookie:retry}'));
+/** @var string $header */
+$header = $translator->t('{core:no_cookie:header}');
+/** @var string $desc */
+$desc = $translator->t('{core:no_cookie:description}');
+/** @var string $retry */
+$retry = $translator->t('{core:no_cookie:retry}');
+
+$t->data['header'] = htmlspecialchars($header);
+$t->data['description'] = htmlspecialchars($desc);
+$t->data['retry'] = htmlspecialchars($retry);
```

Comment:
I'm in favor of that, however we the phpdoc in Template::t tells says it will not be removed until 2.0...
Let's leave this for now.. Removing the deprecated behaviour will be another PR anyway

Reply:
Yep, but I was thinking about removing the documentation for the old usage, not the actual implementation for it. In any case, it's probably better to leave that out for now...